src/Argu/project.json

```diff
@@ -0,0 +1,64 @@
+{
+    "version": "3.0.0-beta02",
+    "buildOptions": {
+        "debugType": "portable",
+        "compilerName": "fsc",
+        "define": [
+            "CORE_CLR"
+        ],
+        "compile": {
+            "includeFiles": [
+                "AssemblyInfo.fs",
+                "Types.fs",
+                "Utils.fs",
+                "ConfigReaders.fs",
+                "UnionArgInfo.fs",
+                "PreCompute.fs",
+                "UnParsers.fs",
+                "ParseResult.fs",
+                "Parsers/Common.fs",
+                "Parsers/Cli.fs",
+                "Parsers/KeyValue.fs",
+                "ArgumentParser.fs"
+            ]
+        }
+    },
+    "tools": {
+        "dotnet-compile-fsc": {
+        "version": "1.0.0-preview2-*",
+        "imports": "dnxcore50"
+        }
+    },
+    "frameworks": {
+        "netstandard1.6": {
+            "buildOptions": {
+                "define": [
+                "NETSTANDARD1_5",
+                "NETSTANDARD1_6"
+                ]
+            },
+            "dependencies": {
+                "System.Xml.XDocument": "4.0.11",
+                "Microsoft.FSharp.Core.netcore": "1.0.0-alpha-160629",
+                "NETStandard.Library": "1.6.0"
+            },
+            "imports": [
+                "portable-net45+win8",
+                "dnxcore50"
+            ]
+        },
+        "net46": { 
```

Comment:
`net46` is not strucly necessary, is nice to check works also on net46, but you can just remove it if you use it only for .net core